### DISCUSSION TITLE
Radiation calculations using SOLWEIG (POI) output

### DISCUSSION THREAD

**Discussion by rarygit** [UMEP-dev/UMEP#364] [Q&A] [answered]
I have some questions regarding SOLWEIG output, any helpful comments appreciated.

(a) Using SOLWEIG output, is this correct for estimating Net Radiation:  Net Radiation = (ldown - lup) + (kdown - kup) 
I am assuming net radiation means surface net solar radiation and influences surface temperature.

For example, ERA-Interim total-sky surface-net solar radiation (SSR) and clear-sky surface-net solar radiation (SSRC) are computed by subtracting the amount of solar radiation reflected by the earth’s surface from the amount of solar radiation reaching a horizontal plane at the surface of the earth (both direct and diffuse).

(b) How do I estimate the incident PAR (photosynthetically active radiation) using output from SOLWEIG?

Reading Weiss, 1984 and Bendix et al. 2010, I understand that you can obtain the PAR fraction from multiplying the global radiation above the canopy by a conversion factor (0.48). I think the value of 0.48 was derived from the research of Weiss, 1984. 
Any other suggestions?
References: 
1. Bendix J, Silva B, Roos K, et al. Model parameterization to simulate and compare the PAR absorption potential of two competing plant species. Int J Biometeorol. 2010;54(3):283-295. doi:10.1007/s00484-009-0279-3
2. Weiss, A., "Partitioning Solar Radiation into Direct and Diffuse, Visible and Near-Infrared Components", (1984). Papers in Natural Resources. 1183. https://digitalcommons.unl.edu/natrespapers/1183

(c) What is the definition for "theoretical value of maximum incoming solar radiation, IO"? 

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
SOLWEIG Output:
kdir	 direct beam solar radiation (from meteorological data) [W m-2]
kdiff	 diffuse component of radiation (from meteorological data) [W m-2]
kdown	 Incoming shortwave radiation [W m-2]
kglobal  global radiation (from meteorological data) [W m-2]
kup	 Outgoing shortwave radiation [W m-2]
ldown	 Incoming longwave radiation [W m-2]
lup	 Outgoing longwave radiation [W m-2]
Tg	 calculated surface temperature [°C]
I0	 theoretical value of maximum incoming solar radiation [W m-2]
CI	 clearness index
Shadow	 Shadow value (Decimal percent)

**Comment by biglimp** [comment, marked as answer]
a) Yes, that is correct. Note that Q* in SOlWEIG is estimated 1.1 meter above ground. Q* in other models and observation dataset can be at different heights.
b) Not really sure but a parameterisation that you propose might work. Not sure if SOLWEIG is the appropriate tool to estimate PAR since it was not designed for this.
c) I0 is estimate from Crawford and Duchon (2010 or 2011 (dont remember). The method calculate extra terrestrial solar radaition and then incudes a number of attenuation factors such as aurosol scattering etc.

**Reply by rarygit** [reply]
Thanks for replying!

Questions a) and c) are answered.

Let's say that b) is problematic.

As an alternative to b), I rephrase the question as follows:
Can I use the Incoming Shortwave Radiation (kdown) from SOLWEIG to represent the global radiation obstructed by a building?
i.e for my urban site's input met data, I want to use obstructed global radiation (kdown), and not the unobstructed global radiation (kglobal). Is this an acceptable fudge in the absence of in-situ measurements? 

![kdown_shadow](https://user-images.githubusercontent.com/16574504/151935017-9bba3be2-99fd-4a91-87c6-21634d653fe2.jpg)

**Reply by biglimp** [reply]
b) Yes, kdown could be used for that purpose I would say.

**Reply by rarygit** [reply]
Thanks for that confirmation, this is very helpful. 
We can take this as answered
